docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+First, we could create an empty DataFrame:
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame()
+0×0 DataFrame
+```
+
+Or, we could call the constructor using keyword arguments to add columns to the DataFrame:
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame(A=1:3, B=5:7, fixed=1)
+3×3 DataFrame
+ Row │ A      B      fixed
+     │ Int64  Int64  Int64
+─────┼─────────────────────
+   1 │     1      5      1
+   2 │     2      6      1
+   3 │     3      7      1
+```
+note in column `:fixed` that scalars get automatically broadcasted.
+
+Now, we will explore how to load a CSV file into into a DataFrame. Unlike Python's Pandas `read_csv`
+the functions in Julia are separated into two modules - `CSV.jl` and `DataFrames.jl`. As the first
+step, you have to declare the libraries you will use. In our case `CSV` and `DataFrames`. In order to
+turn the `CSV.File` to a DataFrame you have to pass it to the `DataFrames.DataFrame` object. You can
+wrap DataFrame around the `CSV.read(path; kwargs)`.
+
+```jldoctest dataframe
+julia> using DataFrames
+
+julia> using CSV
+
+julia> german = CSV.read((joinpath(dirname(pathof(DataFrames)),
+                                 "..", "docs", "src", "assets", "german.csv")),
+                       DataFrame)
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67  male        2  own      NA               little          ⋯
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+```
+
+You can see that Julia representation (unlike python pandas) displays the data type of the column,
+whether it is a `string`, `int` or `float`.
```

Comment:
there are no such types as you list. Either provide a concrete type or unquote and just refer to general concepts like integer, floating point etc.